omod/src/main/webapp/resources/scripts/conditionlist/controllers/conditions.controller.js

```diff
@@ -94,16 +95,24 @@
                 emr.errorAlert("coreapps.conditionui.updateCondition.failure");
                 console.log(emr.message("coreapps.conditionui.updateCondition.failure"), error);
             });
-        }
+        };
+
+        self.formatDate = self.formatDate || function(date) {
+            if (typeof date !== 'undefined' && date !== null) {
+                var locale = $scope.config.locale ? $scope.config.locale : 'en';
+                var format = $scope.config.format ? $scope.config.format : 'DD MMM YYYY';
+                return moment(date).locale(locale).format(format);
+            }
+        };
```

Comment:
This improves upon the previous formatDate function in that it uses the configured javascript date format to be consistent with other pages in the system and is localized.